### PR TITLE
half of model & sql ddl script

### PR DESCRIPTION
**Pull request by arcamelia**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="openjdk-17" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_15" default="true" project-jdk-name="15" project-jdk-type="JavaSDK">
```
Comment (anthonylopes54) [inline .idea/misc.xml:3]: Why did you change this?

Comment (arcamelia) [reply .idea/misc.xml:3]: I must've accidentally clicked something on intelliJ when I opened the project up - we can go ahead and get rid of the edit

Comment (anthonylopes54) [reply .idea/misc.xml:3]: Can you remove it from your PR so that we can merge it?
